src/garage/torch/algos/maml.py

```diff
@@ -164,8 +165,9 @@ def _train_value_function(self, paths):
 
         obs = np.concatenate([path['observations'] for path in paths], axis=0)
         returns = np.concatenate([path['returns'] for path in paths])
-        obs = torch.Tensor(obs)
-        returns = torch.Tensor(returns)
+
+        obs = np_to_torch(obs.astype(np.float32))
```

Comment:
Just curious why we need to use `obs.astype(np.float32)` instead of just `obs`.

Reply:
Oh I missed that. Resolving